ecoscope/io/raster.py

```diff
@@ -204,3 +206,34 @@ def raster_to_gdf(raster_path):
             ],
             crs=src.crs,
         )
+
+
+def grid_to_raster(grid=None, val_column="", out_dir="", raster_name="grid.tif", xlen=5000, ylen=5000, grid_crs=4326):
+    """
+    Save a GeoDataFrame grid to a raster.
+    Does not handle different xlen and ylen
```

Comment:
Does this mean that the xlen and ylen params must be equal? If so why not a single param?

Reply:
I tested different `xlen` and `ylen` and they are handled differently. It may be an outdated comment from the copy-over code.